Azure/azure-cli
# [ACR] Add OCI Artifact Source Context

**Description** 
 
Currently, ACR CLI does not support an [OCI artifact](https://github.com/opencontainers/artifacts) to be passed in as the source context or source location for any of the commands. We are adding support for this new third type of remote context and need to add CLI support for contexts provided of form `oci://myregistry.azurecr.io/artifact`.

**Testing Guide**  

`az acr build -f Dockerfile -r myregistry oci://myregistry.azurecr.io/artifact:v1` This command will take the provided source context (oci://) and use the Dockerfile downloaded from this source context to build. (Need to change registry with own registry. Need to change artifact link with own artifact uploaded to registry)
`az acr run -f acb.yaml -r myregistry oci://myregistry.azurecr.io/artifact:v2` This command will take the provided source context (oci://) and use the acb.yaml downloaded from this source context to run the task. (Need to change registry with own registry. Need to change artifact link with own artifact uploaded to registry)

This checklist is used to make sure that common guidelines for a pull request are followed.
